packages/react-ide-toolbox/src/previews/component-preview.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
   exact?: boolean;
 }
 
-export const ComponentPreview: React.FC<Props> = ({ path, children, setToolsPropsToEdit, exact = false, propsEditInfo }: Props) => {
+export const ComponentPreview: React.FC<Props> = ({ path, children, setToolsPropsToEdit, exact = true, propsEditInfo }: Props) => {
```

Comment:
What is the purpose of this change?

Reply:
It was rebased from master

Reply:
This change is not on master, otherwise it wouldn't be visible in the PR.

Reply:
https://github.com/Haulmont/jmix-frontend/commit/449e259caf99b8da8dfee943fde5a9a629f016be